Source/Scene/BillboardCollection.js

```diff
@@ -1004,6 +1013,8 @@ define([
         boundingVolume.radius += size + offset;
     }
 
+    var writers = [];
```

Comment:
Sorry to bother with local variable names, but can we include `scratch` in this like we do everywhere else?